src/main/java/io/vertx/mqtt/impl/MqttClientImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Red Hat Inc.
+ * Copyright 2016, 2017 Red Hat Inc.
```

Comment:
It is okay to leave 2016 for now. One day @ppatierno will update all headers

Reply:
yes @Sammers21 is right, I have to update all the other files so revert this change please otherwise I'll have some mix :-)

Reply:
Copyright header should always be up to date 😉